documentation/quick_start_qemu/quick_start_qemu_linux.md

```diff
@@ -71,7 +113,11 @@ If you have selected an online package, you can download the package to the pack
 
 ![download the package](figures/ubuntu-update-pkg.png)
 
-### Compile
+#### 4.1 Tips
+
+Before you use the `pkgs` command, you need to type command `source ~/.env/env.sh`. This is a annoying work. We can attach this command as a new line at the end of `~/.bashrc`, which can let you to to use `pkgs` command directly.
```

Comment:
Before using the `pkgs` command, you need to type the command `source ~/.env/env.sh`. This is quite annoying. We can attach this command as a new line at the end of `~/.bashrc`, which can let you to to use `pkgs` command directly.